va/va.h

```diff
@@ -1423,6 +1430,23 @@ typedef union _VAConfigAttribValEncPerBlockControl {
     uint32_t value;
 } VAConfigAttribValEncPerBlockControl;
 
+typedef union _VAConfigAttribValDynamicResolutionChange {
+    struct {
+        /** \brief whether to support dynamic resolution change */
+        uint32_t drc_support             : 1;
+        /** \brief whether need to reset context for drc
+         *
+         *  0: invalid value
+         *  1: need to reset context for drc
+         *  2: could not reset context for drc
```

Comment:
It would be better to indicate "In this case, context resolution will not limit runtime surface resolution."